addons/website_payment/static/src/js/wysiwyg.js

```diff
@@ -0,0 +1,18 @@
+odoo.define("website_payment.wysiwyg", function (require) {
+    "use strict";
+
+const Wysiwyg = require("web_editor.wysiwyg");
+
+Wysiwyg.include({
+    /**
+     * @override
+     */
+    removeLink() {
+        const RequiredButtons = ["s_website_form_send", "s_donation_donate_btn"];
+        if (RequiredButtons.some(className => this.lastElement.classList.contains(className))) {
+            return;
+        }
+        this._super(...arguments);
```

Comment:
We can avoid checking `s_website_form_send` class here as already we are checking that in web_editor wyswyg.
May be this will work,
```suggestion
        if (this.lastElement.classList.contains('s_donation_donate_btn')) {
            return;
        }
        this._super(...arguments);
```